lib/my_app.dart

```diff
@@ -14,24 +16,35 @@ class MyApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    ApiService apiService = ApiService();
-    MovieRemoteRepository movieRepository =
-        MovieRemoteRepository(apiService: apiService);
-    return MaterialApp(
-      theme: ThemeData(
-        useMaterial3: true,
-        colorSchemeSeed: Colors.deepOrange,
-        brightness: Brightness.dark,
-      ),
-      initialRoute: MovieListHome.routeName,
-      routes: {
-        MovieListHome.routeName: (context) => MovieListHome(
-              movieBloc: MovieBloc(
-                  moviesUseCase:
-                      MovieUseCase(movieRepository: movieRepository)),
-            ),
-        MovieHomePage.routeName: (context) => const MovieHomePage(),
-      },
-    );
+    return MultiProvider(
+        providers: [
+          Provider(
+              create: (_) => MovieRemoteRepository(
+                  apiService: Provider.of<ApiService>(context))),
+          Provider(
+              create: (ctx) => MovieUseCase(
```

Comment:
Define the type of the parameters and also be consisten with the name, we suggested to use `context` instead of `ctx`